React/game/src/app/components/App/index.js

```diff
@@ -1,7 +1,33 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-dom';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 
-import Game from '~screen/Game';
+import Game from '@game';
 
-const App = () => <Game />;
+import Login from '@login';
 
-export default App;
+class App extends Component {
+  render() {
+    const isLogin = this.props.isLogin;
+    const validateRoute = () => (isLogin ? <Redirect to="/game" /> : <Login />);
```

Comment:
I think this works for now, but something tells me this code will be problematic once we have more than a `login` screen and a `game` screen

Reply:
:(

Reply:
Yep, this in next PRs need to change